paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentMethodsUI.kt

```diff
@@ -168,100 +175,3 @@ private fun computeItemWidthWhenExceedingMaxWidth(
 
     return ((availableWidth - overallSpacing).value / visibleCards).dp
 }
-
-@Composable
-internal fun PaymentMethodUI(
```

Comment:
This is the code that was moved into the new class. It just didn't track since I moved package, and changed it slightly.